web-console/future-openshift/bulk-actions/bulk-actions.md

```diff
@@ -0,0 +1,26 @@
+# Bulk actions
+
+Bulk actions will be added to a number of pages in the OpenShift console.
+
+
+## Behavior
+* The bulk selector will be the first component in the toolbar on pages where bulk actions are available.
+* Users can click the checkbox in the bulk selector to select all items in the list. If all items are selected, the bulk selector checkbox will be filled with a check and the bulk selector will read 'All items selected'.
+* Users can also choose to select a group of items by clicking the associated checkboxes next to them. The bulk selector checkbox will be filled in with a '-' to indicate that some items are selected, and the bulk selector will display the number of items selected (e.g. '2 selected').
+* Clicking the bulk selector checkbox will deselect all items.
+* If any of the items in the list are selected, the primary action button will be replaced by the actions dropdown.
+
+![1](img/1.png)
+
+![2](img/2.png)
+
+![3](img/3.png)
+
+![4](img/4.png)
+
+## Actions
+* Only specific actions will be available to take on multiple resources. The default bulk actions are 'Add labels', 'Add annotations', and 'Delete Resources'.
+  * Any labels or annotations added will NOT overwrite existing labels or annotations. They will simply be added to the selected resources.
+* Additional actions may be added in future depending on the resource type.
+
+![5](img/5.png)
```

Comment:
Hi @bmignano I was posting earlier most specifically about img/5.png above. Is there a backstory behind why we wouldn't use PF filter chips here? https://www.patternfly.org/v4/documentation/react/components/chipgroup/

Reply:
Ah, sorry for the misunderstanding! Would you recommend using filter chips here? I didn't even think to use them since it's not in the context of a filter. The way it's designed now is it keeps the same exact styling of our labels but adds the ability to delete with the little 'x' at the end. The visual consistency is nice since users are adding labels and they also look like labels.

Reply:
Thanks for the quick response! I am following up on this and will get back soon :)

Reply:
Hi @bmignano I spoke with Kyle, and he thinks this would be a good one to take a quick look at together. Could you hang on at the end of the sync today and we'll discuss?

Reply:
@rachael-phillips @bmignano what is the status of this discussion with the PF side? 
Brie - let me know if this is in good shape for review!

Reply:
@beanh66 Sounds like we're all set from the PatternFly side! I think it's ready for review 👍